package.json

```diff
@@ -6,7 +6,8 @@
   "main": "scalardl-node-client-sdk.js",
   "license": "AGPL-3.0-or-later",
   "engines": {
-    "node": ">=16.0.0"
+    "node": ">=14 <17",
+    "npm": ">=7"
```

Comment:
Tell the developers that this package can be run on
- version 14 (maintenance LTS)
- version 15 (EOL)
- version 16 (active LTS)

The NPM needs to be higher than 7 because we are using package-lock.json version 2.